packages/eslint-plugin-next/src/rules/no-img-element.ts

```diff
@@ -31,7 +31,7 @@ export = defineRule({
 
         context.report({
           node,
-          message: `Using \`<img>\` could result in slower LCP and higher bandwidth. Use \`<Image />\` from \`next/image\` instead to utilize Image Optimization. See: ${url}`,
+          message: `Using \`<img>\` could result in slower LCP and higher bandwidth due to image sizes. Consider using \`<Image />\` from \`next/image\` to automatically optimize images. Note: when self-hosting, Image Optimization runs on your Next.js server. If using an external loader, additional usage or cost could be incurred, depending on your provider. See: ${url}`,
```

Comment:
```suggestion
          message: `Using \`<img>\` could result in slower LCP and higher bandwidth due to image size. Consider using \`<Image />\` from \`next/image\` to automatically optimize images. Note: check with your provider to see the price of bandwidth and the price of image optimization to see how this may impact your bill. See: ${url}`,
```